docs/usage.md

```diff
@@ -224,6 +234,80 @@ Key notes:
+| - | - | - |
+| id | int | A unique id that is assigned to an attempt at a question in a pset interactive, same as `id` in `ib_pset_problem_attempts.csv` |
+| impression_id | UUID | A unique identifier that can be used to associate events in a single user impression, same as `impression_id` in `content_loads.csv` and `ib_pset_problem_attempts.csv` |
+| user_uuid | UUID | User UUID |
+| course_id | int | Course ID |
+| content_page_id | UUID | Content page UUID, same as `content_id` in `course_contents.csv` and `ib_pset_problem_attempts.csv` |
+| section | str | Section name of the associated content page |
+| activity | str | Activity name of the associated content page |
+| lesson_page | str | Lesson page name of the associated content page |
+| pset_id | UUID | Problem set ID, same as `pset_content_id` in `ib_pset_problem_attempts.csv` and `pset_id` in `ib_pset_problems.csv` |
+| pset_problem_id | UUID | Problem ID, same as `pset_problem_content_id` in `ib_pset_problem_attempts.csv` and `id` in `ib_pset_problems.csv` |
+| timestamp | int | A Unix timestamp value that reflects when the grade was created for this problem attempt (milliseconds that have elapsed since 00:00:00 UTC on January 1, 1970) |
+| variant | str | Variant name |
+| problem_type | str | The type of problem (one of 'input', 'dropdown', 'multiselect', or 'multiplechoice') |
+| problem_content | str | Problem content, same as `content` in `ib_pset_problems.csv` |
+| problem_solution | str | Correct solution for problem, same as `solution` in `ib_pset_problems.csv`|
+| solution_options | str | Solution options for problem, same as `solution_options` in `ib_pset_problems.csv` |
+| problem_response | str or str[] when `problem_type` is `multiselect` | User response to the problem, same as `response` in `ib_pset_problem_attempts.csv` |
+| is_correct | bool | Whether the response was correct, same as `correct` in `ib_pset_problem_attempts.csv` |
+| attempt_number | int | Attempt count for the problem, same as `attempt` in `ib_pset_problem_attempts.csv` |
+| is_final_attempt | bool | Whether the attempt was the final allowed attempt for the problem, same as `final_attempt` in `ib_pset_problem_attempts.csv` |
+
+## `view_quiz.csv`
+
+| Column | Type | Notes |
+| - | - | - |
+| id | int | A unique id that is assigned to an attempt at a question in a quiz, same as `id` in `quiz_attempt_multichoice_responses.csv` |
+| attempt_id | int | Attempt ID of a quiz, same as `id` in `quiz_attempts.csv` and `attempt_id` in `quiz_attempt_multichoice_responses.csv` |
+| quiz_id | int | Quiz ID, same as `assessment_id` in `quiz_attempts.csv` and `id` in `assessments.csv` |
+| quiz_name | str | The user friendly name associated with this quiz, same as `assessment_name` in `assessments.csv`|
+| user_uuid | UUID | User UUID |
+| course_id | int | Course ID |
+| quiz_attempt_number | int | The attempt number for the user in this quiz, same as `attempt_number` in `quiz_attempts.csv` |
```

Comment:
Let's call this `quiz_attempt_attempt_number` (yeah, I realize that is redundant, but I'm erroring on the side of consistency)